composer.json

```diff
@@ -21,7 +21,7 @@
 		"kdyby/translation": "~2.2",
 		"kdyby/console": "^2.5",
 		"sendgrid/sendgrid": "^4.0",
-		"kdyby/facebook": "~2.0",
+		"kdyby/facebook": "2.0.x-dev",
```

Comment:
because of this fix for Nette 2.4 https://github.com/Kdyby/Facebook/commit/a0752c43ed94b74663b975f240166d9aaa85ccca